org.freedesktop.Sdk.Extension.openjfx.yaml

```diff
@@ -0,0 +1,43 @@
+id: org.freedesktop.Sdk.Extension.openjfx
+branch: '23.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+build-extension: true
+sdk: org.freedesktop.Sdk
+separate-locales: false
+appstream-compose: false
+build-options:
+  no-debuginfo: true
+  prefix: /usr/lib/sdk/openjfx
+
+modules:
+  - name: javafx_jmods
+    buildsystem: simple
+    sources:
+      - type: file
```

Comment:
```suggestion
      - type: archive
```

Reply:
This is by design since I'm only interested in a few javafx.jmods but of course I can let archive extract everything and move the wanted files.

Reply:
Ok then it is fine